features/autoconsent.json

```diff
@@ -258,6 +258,10 @@
         {
             "domain": "www.thrifty.com",
             "reason": "Page loads with gray overlay over content."
+        },
+        {
+            "domain": "ngrok.com",
+            "reason": "some transparent overlay prevents elements to be clicked upon"
```

Comment:
@icodebyamanda I think this is a wider problem with Ketch CMP. Let's just disable the whole rule in "disabledCMPs" below?